res/css/structures/_GroupView.scss

```diff
@@ -368,6 +368,61 @@ limitations under the License.
     padding: 40px 20px;
 }
 
+.mx_GroupView_spaceUpgradePrompt {
+    padding: 16px 50px;
+    background-color: $header-panel-bg-color;
+    border-radius: 8px;
+    max-width: 632px;
+    font-size: $font-15px;
+    line-height: $font-24px;
+    margin-top: 24px;
+    position: relative;
+
+    > h2 {
+        font-size: inherit;
+        font-weight: $font-semi-bold;
+    }
+
+    > p, h2 {
+        margin: 0;
+    }
+
+    &::before {
```

Comment:
is there no way to make this declaration reusable? this pattern is repeated several times

Reply:
only 3 of the attributes are identical between all instances (content, position, mask-repeat), everything else is specific to the usage